mint-auditor/src/bin/main.rs

```diff
@@ -257,6 +271,35 @@ fn cmd_get_block_audit_data(
     .expect("db transaction failed");
 }
 
+// start the http server
+async fn cmd_start_http_server(mint_auditor_db_path: PathBuf, logger: Logger) {
+    let mint_auditor_db = MintAuditorDb::new_from_path(
+        &mint_auditor_db_path.into_os_string().into_string().unwrap(),
+        DB_POOL_SIZE,
+        logger.clone(),
+    )
+    .expect("Could not open mint auditor DB");
+
+    if let Err(e) = rocket::build()
+        .manage(mint_auditor_db)
+        .mount(
+            "/",
+            routes![
+                routes::index,
+                routes::get_counters,
+                routes::get_block_audit_data,
+                routes::get_last_block_audit_data,
+            ],
+        )
+        .launch()
+        .await
+    {
+        println!("Whoops! Rocket didn't launch!");
+        // We drop the error to get a Rocket-formatted panic.
+        drop(e);
+    }
```

Comment:
IIUC this is equivalent to calling `expect()`
```suggestion
    rocket::build()
        .manage(mint_auditor_db)
        .mount(
            "/",
            routes![
                routes::index,
                routes::get_counters,
                routes::get_block_audit_data,
                routes::get_last_block_audit_data,
            ],
        )
        .launch()
        .await
        .expect("Whoops! Rocket didn't launch!");
```

Reply:
If i use `expect` I get an error: `unused rocket must be used`. Also, I pulled this block for starting rocket directly from the rocket docs.